include/boost/geometry/core/point_type.hpp

```diff
@@ -92,6 +92,16 @@ struct point_type<ring_tag, Ring>
     typedef typename boost::range_value<Ring>::type type;
 };
 
+// Specialization for PolyhedralSurface: the point-type is the point-type of its rings
```

Comment:
`Of its rings` -> please change to `of its ...` (whatever is our name for the surfaces making up the polyhedron)

Reply:
poly_ring_type is a good name i guess